src/taipy/core/_scheduler/_dispatcher/_job_dispatcher.py

```diff
@@ -41,6 +42,7 @@ def __init__(self, scheduler: _AbstractScheduler):
         threading.Thread.__init__(self, name="Thread-Taipy-JobDispatcher")
         self.daemon = True
         self.scheduler = scheduler
+        Config.block_update()
```

Comment:
This is a minor nonblocking remark.
I don't see any use case for now, but it feels more logical to me to:
- call `Config.block_update()` in the start method instead of the init.
- call `Config.unblock_update()` at the end of the run method, just before returning.

What do you think?

Reply:
On Development mode, there will be no call to the `start` or `run` method though.

I did implemented this in some of the previous version, which make much more sense on standalone mode.

Reply:
ok thx.